Makefile.PL

```diff
@@ -52,6 +52,7 @@ if ($Config{myuname} =~ /sunos|solaris/i) {
 } elsif ($Config{myuname} =~ /darwin/ ) {
   $args{LDDLFLAGS} = openssl_lib_paths() . " " . $Config{lddlflags};
 } else {
+  $args{LDDLFLAGS} = openssl_lib_paths() . " " . $Config{lddlflags};
```

Comment:
Please replace double quotes with single quotes. 

Too many years with Perl::Critic, has made me allergic to double quotes